From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: wisemanjr4 <mitsuking2003@gmail.com>
Date: Tue, 24 Feb 2026 18:42:52 +0900
Subject: [PATCH] Rebrand to SuperVertex


diff --git a/src/main/java/net/minecraft/server/MinecraftServer.java b/src/main/java/net/minecraft/server/MinecraftServer.java
index a358a821e2ed71fdb2e85c71b75fda00fdad7f9b..0c6482994ffd37ff83bc619206024395d393122e 100644
--- a/src/main/java/net/minecraft/server/MinecraftServer.java
+++ b/src/main/java/net/minecraft/server/MinecraftServer.java
@@ -932,7 +932,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
         shutdownThread = Thread.currentThread();
         org.spigotmc.WatchdogThread.doStop(); // Paper
         if (!isSameThread()) {
-            MinecraftServer.LOGGER.info("Stopping main thread (Ignore any thread death message you see! - DO NOT REPORT THREAD DEATH TO PLAZMA)"); // Purpur // Plazma
+            MinecraftServer.LOGGER.info("Stopping main thread (Ignore any thread death message you see! - DO NOT REPORT THREAD DEATH TO SUPERVERTEX)"); // Purpur // Plazma // SuperVertex
             while (this.getRunningThread().isAlive()) {
                 this.getRunningThread().stop();
                 try {
diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
index 84fb7f6cdad5e7a5389e2469bfd6fb961cd1b299..a44b756630d5bb77d812bfc7a8ae697ce95403fb 100644
--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
@@ -262,7 +262,7 @@ import javax.annotation.Nullable; // Paper
 import javax.annotation.Nonnull; // Paper
 
 public final class CraftServer implements Server {
-    private final String serverName = "Plazma"; // Paper // Pufferfish // Purpur // Plazma
+    private final String serverName = "SuperVertex"; // Paper // Pufferfish // Purpur // Plazma // SuperVertex
     private final String serverVersion;
     private final String bukkitVersion = Versioning.getBukkitVersion();
     private final Logger logger = Logger.getLogger("Minecraft");
diff --git a/src/main/java/org/purpurmc/purpur/PurpurConfig.java b/src/main/java/org/purpurmc/purpur/PurpurConfig.java
index 072cc011abded538ae643b36cfb1f861d564bcec..1537fda9f1279ea4a01bcf8de38bd741b37dc834 100644
--- a/src/main/java/org/purpurmc/purpur/PurpurConfig.java
+++ b/src/main/java/org/purpurmc/purpur/PurpurConfig.java
@@ -232,7 +232,7 @@ public class PurpurConfig {
         deathMessageOnlyBroadcastToAffectedPlayer = getBoolean("settings.broadcasts.death.only-broadcast-to-affected-player", deathMessageOnlyBroadcastToAffectedPlayer);
     }
 
-    public static String serverModName = "Plazma"; // Plazma
+    public static String serverModName = "SuperVertex"; // Plazma // SuperVertex
     private static void serverModName() {
         serverModName = getString("settings.server-mod-name", serverModName);
     }
